src/controllers/apiController.ts

```diff
@@ -0,0 +1,364 @@
+    req.throwOn(401, ApiError, 'Unauthorized');
+    req.throwOn(403, ApiError, 'Access Denied');
+    req.throwOn(404, ApiError, 'Not Found');
+    return req.call(requestOptions);
+  }
+
+  /**
+   * Remove a participant from a session
+   *
+   * This will automatically remove any subscriptions the participant has associated with this session
+   *
+   *
+   * @param accountId     Account ID
+   * @param participantId Participant ID
+   * @param sessionId     Session ID
+   * @return Response from the API call
+   */
+  async removeParticipantFromSession(
+    accountId: string,
+    participantId: string,
+    sessionId: string,
+    requestOptions?: RequestOptions
+  ): Promise<ApiResponse<void>> {
+    const req = this.createRequest('DELETE');
+    req.baseUrl('WebRtcDefault');
+    const mapped = req.prepareArgs({
+      accountId: [accountId, string()],
+      participantId: [participantId, string()],
+      sessionId: [sessionId, string()],
+    });
+    req.appendTemplatePath`/accounts/${mapped.accountId}/sessions/${mapped.participantId}/participants/${mapped.sessionId}`;
+    req.throwOn(401, ApiError, 'Unauthorized');
+    req.throwOn(403, ApiError, 'Access Denied');
+    req.throwOn(404, ApiError, 'Not Found');
+    return req.call(requestOptions);
+  }
+
+  /**
+   * Get a participant's subscriptions
+   *
+   * @param accountId     Account ID
+   * @param participantId Participant ID
+   * @param sessionId     Session ID
+   * @return Response from the API call
+   */
+  async getParticipantSubscriptions(
+    accountId: string,
+    participantId: string,
+    sessionId: string,
+    requestOptions?: RequestOptions
+  ): Promise<ApiResponse<Subscriptions>> {
+    const req = this.createRequest('GET');
+    req.baseUrl('WebRtcDefault');
+    const mapped = req.prepareArgs({
+      accountId: [accountId, string()],
+      participantId: [participantId, string()],
+      sessionId: [sessionId, string()],
+    });
+    req.appendTemplatePath`/accounts/${mapped.accountId}/sessions/${mapped.participantId}/participants/${mapped.sessionId}/subscriptions`;
+    req.throwOn(401, ApiError, 'Unauthorized');
+    req.throwOn(403, ApiError, 'Access Denied');
+    req.throwOn(404, ApiError, 'Not Found');
+    return req.callAsJson(subscriptionsSchema, requestOptions);
+  }
+
+  /**
+   * Update a participant's subscriptions
+   *
+   * This is a full update that will replace the participant's subscriptions. First call
+   * `getParticipantSubscriptions` if you need the current subscriptions. Call this function with no
+   * `Subscriptions` object to remove all subscriptions
+   *
+   *
+   * @param accountId     Account ID
+   * @param participantId Participant ID
+   * @param sessionId     Session ID
+   * @param body          Initial state
+   * @return Response from the API call
+   */
+  async updateParticipantSubscriptions(
+    accountId: string,
+    participantId: string,
+    sessionId: string,
```

Comment:
Soo funny story, we never caught this but we accidentally defined the order of these incorrectly in the OpenAPI spec. Since this is a major version change should we change this with the release too?

Reply:
I'm more than happy to do it.. I think I know the fix

Reply:
Yeah go for it

Reply:
I think it'll be important to call it out in the release notes

Reply:
@jmulford-bw The remove and get functions are backwards too

Reply:
Yes 100%. This is a breaking change that doesn't cause a syntax error :(

Reply:
Got it on the other functions too. Thanks for double checking